Tella/Domain/Entity/Uwazi/Template/UwaziTemplateRow.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UwaziTemplateRow.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/8/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+// TODO: Removed unwanted variables when done with entity flow no removed now due to confusion on what will be need in future implementation
```

Comment:
In this case we need to remove the conversion `toDomain` from Uwazi template models, because in the current implementation the conversion is not implemented properly